src/server/procedures/myForest/projectList.ts

```
@@ -34,19 +37,27 @@ export const projectListsProcedure = procedure.query(async () => {
-      allowDonations: project.allowDonations ? true : false,
-    };
-    dictionary[project.guid] = newProject;
-    return dictionary;
-  }, {});
+    // Convert projects to an object with index as guid, and value as project
+    const projectsDictionary = projects.reduce<{
+      [key: string]: MyForestProject;
+    }>((dictionary, project) => {
+      const newProject: MyForestProject = {
+        ...project,
+        allowDonations: project.allowDonations ? true : false,
```

Biome (1.9.4)

[error] Unnecessary use of boolean literals in conditional expression.

Simplify your code by directly assigning the result without using a ternary operator.
If your goal is negation, you may use the logical NOT (!) or double NOT (!!) operator for clearer and concise code.
 Check for more details about NOT operator.
Unsafe fix: Remove the conditional expression with


(lint/complexity/noUselessTernary)